--- tiopenmax-gsmhrenc-3.22.orig/audio/src/openmax_il/gsmhr_enc/tests/Makefile	2008-07-25 14:22:33.000000000 -0500
+++ tiopenmax-gsmhrenc-3.22/audio/src/openmax_il/gsmhr_enc/tests/Makefile	2009-05-25 14:08:56.000000000 -0500
@@ -41,7 +41,6 @@
 endif
 
 
-OMXCORE_LIB=$(OMXLIBDIR)/libOMX_Core.so
 OMXRMPROXY_LIB=$(OMXLIBDIR)/libOMX_ResourceManagerProxy.so
 COMPONENT_TEST=GSMHrEncTest_common
 
@@ -49,7 +48,7 @@
 
 all install:: $(COMPONENT_TARGET)
 
-omxcore $(COMPONENT_TARGET): $(OMXTESTDIR) $(COMPONENT_TEST) $(OMXLIBDIR) $(OMXCORE_LIB)  
+$(COMPONENT_TARGET): $(OMXTESTDIR) $(COMPONENT_TEST) $(OMXLIBDIR)
 	@echo "Installing $(COMPONENT_TEST)"
 	cp -f $(COMPONENT_TEST) $(COMPONENT_TARGET)
 
