diff -Naur linux.orig/audio/src/openmax_il/g729_dec/src/OMX_G729Dec_Utils.c linux/audio/src/openmax_il/g729_dec/src/OMX_G729Dec_Utils.c
--- linux.orig/audio/src/openmax_il/g729_dec/src/OMX_G729Dec_Utils.c	2009-03-10 14:29:39.000000000 -0600
+++ linux/audio/src/openmax_il/g729_dec/src/OMX_G729Dec_Utils.c	2009-04-17 14:10:42.000000000 -0500
@@ -1871,7 +1871,6 @@
     } 
     else if (event == EMMCodecStrmCtrlAck) 
     {
-        LCML_DPRINT("%d :: GOT MESSAGE USN_DSPACK_STRMCTRL ----\n",__LINE__);
         if (args[1] == (void *)USN_STRMCMD_FLUSH)
         {
             G729DEC_DPRINT("revceived USN_STRMCMD_FLUSH\n");
