diff -Naur linux.orig/audio/src/openmax_il/g722_enc/src/OMX_G722Enc_Utils.c linux/audio/src/openmax_il/g722_enc/src/OMX_G722Enc_Utils.c
--- linux.orig/audio/src/openmax_il/g722_enc/src/OMX_G722Enc_Utils.c	2009-03-10 14:29:39.000000000 -0600
+++ linux/audio/src/openmax_il/g722_enc/src/OMX_G722Enc_Utils.c	2009-04-17 14:10:42.000000000 -0500
@@ -1640,9 +1640,7 @@
 		}
 		pComponentPrivate_CC->bNoIdleOnStop= OMX_FALSE;
     }
-    else if(event == EMMCodecAlgCtrlAck) {
-        LCML_DPRINT ("GOT MESSAGE USN_DSPACK_ALGCTRL \n");
-    }
+
     else if (event == EMMCodecDspError) {
 		if(((int)args[4] == 1) && ((int)args[5] == 0x500)) {
 			G722ENC_DPRINT ("%d :: Comp: Inside the G722ENC_LCML_Callback: USN_DSPMSG_ERROR \n", __LINE__);
@@ -1676,7 +1674,6 @@
         }
     }
     else if (event == EMMCodecStrmCtrlAck) {
-        LCML_DPRINT("%d :: GOT MESSAGE USN_DSPACK_STRMCTRL ----\n",__LINE__);
 		pComponentPrivate_CC->bStreamCtrlCalled = 1;
 	
     }
