diff -Naur linux.orig/audio/src/openmax_il/aac_enc/tests/AacEncTest.c linux/audio/src/openmax_il/aac_enc/tests/AacEncTest.c
--- linux.orig/audio/src/openmax_il/aac_enc/tests/AacEncTest.c	2009-03-10 14:29:03.000000000 -0600
+++ linux/audio/src/openmax_il/aac_enc/tests/AacEncTest.c	2009-04-20 20:30:42.000000000 -0500
@@ -56,7 +56,7 @@
 #define APP_INFO
 #define APP_ERROR
 #define DASF
-#undef USE_BUFFER
+#define USE_BUFFER
 #undef AACENC_DEBUGMEM 
 
 /*#define GT_PERFM  *//*Defines the Performance and measurements mode*/
@@ -575,8 +575,8 @@
 
 	OMX_U32	streamId = 0;
 #ifdef USE_BUFFER
-    OMX_U8* pInputBuffer[5] = NULL;
-    OMX_U8* pOutputBuffer[5] = NULL;
+    OMX_U8* pInputBuffer[5] = {NULL};
+    OMX_U8* pOutputBuffer[5] = {NULL};
 #endif
 
     fd_set rfds;
