diff -Naur linux.orig/audio/src/openmax_il/aac_dec/tests/AacDecTest.c linux/audio/src/openmax_il/aac_dec/tests/AacDecTest.c
--- linux.orig/audio/src/openmax_il/aac_dec/tests/AacDecTest.c	2009-03-10 14:26:48.000000000 -0600
+++ linux/audio/src/openmax_il/aac_dec/tests/AacDecTest.c	2009-04-20 20:24:59.000000000 -0500
@@ -78,7 +78,7 @@
 
 #undef APP_DEBUG 
 
-#undef USE_BUFFER 
+#define USE_BUFFER 
 
 /*For timestamp and tickcount*/
 #undef APP_TIME_TIC_DEBUG
@@ -387,8 +387,8 @@
     OMX_S16 numOfOutputBuffer = 0;
     OMX_INDEXTYPE index;
 #ifdef USE_BUFFER
-    OMX_U8* pInputBuffer[10] = NULL;
-    OMX_U8* pOutputBuffer[10] = NULL;
+    OMX_U8* pInputBuffer[10] = {NULL};
+    OMX_U8* pOutputBuffer[10] = {NULL};
 #endif
     struct timeval tv;
     int retval = 0, j = 0, nProfile = 0, i = 0, naacformat=0;
