diff -Naur linux.orig/audio/src/openmax_il/nbamr_dec/tests/AmrDecTest.c linux/audio/src/openmax_il/nbamr_dec/tests/AmrDecTest.c
--- linux.orig/audio/src/openmax_il/nbamr_dec/tests/AmrDecTest.c	2009-03-10 14:36:19.000000000 -0600
+++ linux/audio/src/openmax_il/nbamr_dec/tests/AmrDecTest.c	2009-04-20 22:39:05.000000000 -0500
@@ -48,10 +48,6 @@
 
 OMX_S16 numInputBuffers = 0;
 OMX_S16 numOutputBuffers = 0;
-#ifdef USE_BUFFER
-    OMX_U8* pInputBuffer[10] = {NULL};
-    OMX_U8* pOutputBuffer[10] ={NULL};
-#endif
 
 OMX_BUFFERHEADERTYPE* pInputBufferHeader[10] = {NULL};
 OMX_BUFFERHEADERTYPE* pOutputBufferHeader[10] = {NULL};
@@ -93,7 +89,11 @@
 #define DASF 1
 
 /*#define USE_BUFFER*/
-#undef USE_BUFFER
+#define USE_BUFFER
+#ifdef USE_BUFFER
+    OMX_U8* pInputBuffer[10] = {NULL};
+    OMX_U8* pOutputBuffer[10] ={NULL};
+#endif
 
 #ifdef APP_DEBUG
     #define APP_DPRINT(...)    fprintf(stderr,__VA_ARGS__)
@@ -2280,7 +2280,7 @@
                      close(nbamrEvent_Pipe[1]);
 
 
-		if (fileOut != NULL)	/* Could have been closed  previously */ */
+		if (fileOut != NULL)	/* Could have been closed  previously */
 		{
 			fclose(fileOut);
 			fileOut=NULL;
