diff -Naur linux.orig/audio/src/openmax_il/nbamr_enc/tests/AmrEncTest.c linux/audio/src/openmax_il/nbamr_enc/tests/AmrEncTest.c
--- linux.orig/audio/src/openmax_il/nbamr_enc/tests/AmrEncTest.c	2009-03-10 14:36:38.000000000 -0600
+++ linux/audio/src/openmax_il/nbamr_enc/tests/AmrEncTest.c	2009-04-20 20:46:58.000000000 -0500
@@ -146,7 +146,7 @@
 
 #undef APP_MEMCHECK
 
-#undef USE_BUFFER
+#define USE_BUFFER
 /*#define USE_BUFFER*/
 
 #define STRESS_TEST_ITERATIONS 20
