diff -Naur linux.orig/audio/src/openmax_il/wbamr_dec/tests/WBAmrDecTest.c linux/audio/src/openmax_il/wbamr_dec/tests/WBAmrDecTest.c
--- linux.orig/audio/src/openmax_il/wbamr_dec/tests/WBAmrDecTest.c	2009-03-10 14:38:29.000000000 -0600
+++ linux/audio/src/openmax_il/wbamr_dec/tests/WBAmrDecTest.c	2009-04-20 21:10:11.000000000 -0500
@@ -81,7 +81,7 @@
 #undef APP_DEBUG
 #undef APP_MEMCHECK
 #define DASF 1
-#undef USE_BUFFER
+#define USE_BUFFER
 
 #ifdef APP_DEBUG
     #define APP_DPRINT(...)    fprintf(stderr,__VA_ARGS__)
