diff -Naur linux.orig/audio/src/openmax_il/armaac_dec/tests/ArmAacDecTest.c linux/audio/src/openmax_il/armaac_dec/tests/ArmAacDecTest.c
--- linux.orig/audio/src/openmax_il/armaac_dec/tests/ArmAacDecTest.c	2009-03-10 14:29:19.000000000 -0600
+++ linux/audio/src/openmax_il/armaac_dec/tests/ArmAacDecTest.c	2009-04-20 20:39:01.000000000 -0500
@@ -83,7 +83,7 @@
 
 #undef	APP_DEBUG 
 
-#undef USE_BUFFER 
+#define USE_BUFFER 
 
 /*For timestamp and tickcount*/
 #undef APP_TIME_TIC_DEBUG
