--- tiopenmax-rmrvparser-3.22.orig/video/src/openmax_il/rm_rvparser/src/Makefile	2008-05-05 07:20:02.000000000 -0500
+++ tiopenmax-rmrvparser-3.22/video/src/openmax_il/rm_rvparser/src/Makefile	2009-05-26 19:58:17.000000000 -0500
@@ -41,7 +41,7 @@
 
 all install:: $(COMPONENT_TARGET)
 
-omxcore $(COMPONENT_TARGET): $(COMPONENT_LIB) $(OMXLIBDIR) $(OMXCORE_LIB)
+$(COMPONENT_TARGET): $(COMPONENT_LIB) $(OMXLIBDIR)
 	@echo "Installing $(COMPONENT_LIB)"
 	cp -f $(COMPONENT_LIB) $(COMPONENT_TARGET)
 
