--- tiopenmax-khronos-3.22.orig/system/src/openmax_il/khronos/makefile	2008-10-08 17:15:37.000000000 -0500
+++ tiopenmax-khronos-3.22/system/src/openmax_il/khronos/makefile	2009-05-27 19:59:58.000000000 -0500
@@ -147,13 +147,13 @@
 all: $(PROGRAM)  
 
 $(PROGRAM):	$(OBJS) $(HDRS)
-		$(CC) $(CFLAGS) $(OBJS) -o $(PROGRAM) $(LIBS)\
-		-fpic -pipe -DOMAP_2430 -DOMX_DEBUG\
-		-I$(BRIDGEINCLUDEDIR)\
-	       	-I$(PKGDIR)/target/include/omx\
-		-L$(PKGDIR)/target/lib\
+		$(CC) $(CFLAGS) $(OBJS) -o $(PROGRAM) $(LIBS) \
+		-fpic -pipe -DOMAP_2430 -DOMX_DEBUG \
+		-I$(BRIDGEINCLUDEDIR) \
+	       	-I$(PKGDIR)/omx \
+		-L$(PKGDIR)/lib \
 	        -ldl -lpthread -lOMX_Core -lbridge
-		cp -f $(PROGRAM) $(PKGDIR)/target/omx
+		cp -f $(PROGRAM) $(PKGDIR)/omx
 
@@ -160,4 +160,4 @@
 	rm -f $(PROGRAM) $(PROGRAM_FILE) $(TESTHARNESS)/BaseProfileTests/*.o *.o *.so
 	rm -f $(PROGRAM) $(PROGRAM_FILE) $(TESTHARNESS)/InteropProfileTests/*.o *.o *.so
 	rm -f $(PROGRAM) $(PROGRAM_FILE) $(TESTHARNESS)/StandardComponentTests/*.o *.o *.so
-	rm -f $(PROGRAM) $(PKGDIR)/target/omx/conftest		
+	rm -f $(PROGRAM) $(PKGDIR)/omx/conftest		
